src/components/Dashboard/Capacity/CapacityItem.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { DonutChart } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { InlineLoading } from '../../Loading';
+
+const pfGetUtilizationDonutTooltipContents = d =>
+  `<span class="donut-tooltip-pf" style="white-space: nowrap;">${Math.round(d[0].ratio * 1000) / 10} ${
+    d[0].name
+  }</span>`;
+
+const LoadingCapacityItem = ({ idSuffix, title, LoadingComponent }) => (
+  <div className="kubevirt-capacity__item">
+    <h2 className="kubevirt-capacity__item-title">{title}</h2>
+    <h6>
+      <LoadingComponent />
+    </h6>
+    <div>
+      <DonutChart
+        id={`donut-chart-${idSuffix}`}
+        size={{ width: 150, height: 150 }}
+        data={{
+          unload: true,
+          columns: [['Used', 0], ['Available', 100]],
+          order: null,
+        }}
+        unloadBeforeLoad
+        title={{
+          primary: 'Loading',
+          secondary: 'Capacity',
+        }}
+      />
+    </div>
+  </div>
+);
+
+LoadingCapacityItem.propTypes = {
+  idSuffix: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  LoadingComponent: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+};
+
+export const CapacityItem = ({ idSuffix, title, used, total, unit, formatValue, LoadingComponent }) => {
+  if (!Number.isNaN(parseFloat(used)) && !Number.isNaN(parseFloat(total))) {
```

Comment:
this component probably shouldn't do the parsing and expect numbers instead. We could maybe do this in `getCapacityStats`. Then we can check for null here

Reply:
parsing moved to `getCapacityStats` and `CapacityItem` is checking for null